controllers/applicationvolumereplication_controller.go

```diff
@@ -125,6 +126,42 @@ func (r *ApplicationVolumeReplicationReconciler) Reconcile(ctx context.Context,
 	return ctrl.Result{Requeue: requeue}, nil
 }
 
+func IsManifestInAppliedState(mw *ocmworkv1.ManifestWork) bool {
+	isApplied := false
+	conditions := mw.Status.Conditions
+
+	if len(conditions) > 0 {
+		// sort conditions by timestamp. Index 0 = most recent
+		sort.Slice(conditions, func(a, b int) bool {
+			return conditions[b].LastTransitionTime.Before(&conditions[a].LastTransitionTime)
+		})
+
+		mostRecentTimestamp := conditions[0].LastTransitionTime
+
+		// loop through conditions until not in the most recent one anymore
+		for index := range conditions {
+			// only index 1+ needs exit check
+			if index > 0 {
+				timestamp := conditions[index].LastTransitionTime
+
+				// only evaluate conditions on most recent timestamp
+				if timestamp != mostRecentTimestamp {
+					break
+				}
+			}
+
+			// check if state is applied
+			if conditions[index].Type == "Applied" {
```

Comment:
Use `ocmworkv1.WorkApplied` instead of the string "Applied"

Reply:
Good point. I switched out all the hard-coded strings in `IsManifestWorkInAppliedState` and its tests to use Types on `d41ab23fee18f7e5e248be22830fc12385e9075e`